start.py

```diff
@@ -897,6 +897,51 @@ def _bootstrap_from_code(use_version, use_staging):
     return version_path
 
 
+def _boot_validate_versions(use_version, local_version):
+    _print(f">>> Validating version [ {use_version} ]")
+    openpype_versions = bootstrap.find_openpype(include_zips=True,
+                                                staging=True)
+    openpype_versions += bootstrap.find_openpype(include_zips=True,
+                                                 staging=False)
+    v: OpenPypeVersion
+    found = [v for v in openpype_versions if str(v) == use_version]
+    if not found:
+        _print(f"!!! Version [ {use_version} ] not found.")
+        list_versions(openpype_versions, local_version)
+        sys.exit(1)
+
+    # print result
+    result = bootstrap.validate_openpype_version(
+        bootstrap.get_version_path_from_list(
+            use_version, openpype_versions))
+
+    _print("{}{}".format(
+        ">>> " if result[0] else "!!! ",
+        bootstrap.validate_openpype_version(
+            bootstrap.get_version_path_from_list(
+                use_version, openpype_versions)
+        )[1])
```

Comment:
Isn't this the same?
```suggestion
    _print("{}{}".format(
        ">>> " if result[0] else "!!! ", result[1]
        )
```